test/common/http/conn_manager_utility_test.cc

```diff
@@ -68,7 +68,8 @@ TEST_F(ConnectionManagerUtilityTest, ShouldTraceRequest) {
         Http::AccessLog::ResponseFlag::NoHealthyUpstream,
         Http::AccessLog::ResponseFlag::UpstreamRequestTimeout,
         Http::AccessLog::ResponseFlag::UpstreamOverflow,
-        Http::AccessLog::ResponseFlag::FaultInjected, Http::AccessLog::ResponseFlag::NoRouteFound};
+        Http::AccessLog::ResponseFlag::DelayInjected, Http::AccessLog::ResponseFlag::FaultInjected,
```

Comment:
when egress happens though Envoy1 from Service2 (Service1-> Envoy1 -> Envoy2 -> Service2) and FI injected there will not be proxying request to the Envoy2, we need to have tracing in place in Envoy1 where FI was generated.